Sources/PostgresNIO/Data/PostgresDataType.swift

```diff
@@ -1,11 +1,14 @@
-/// The format code being used for the field.
-/// Currently will be zero (text) or one (binary).
-/// In a RowDescription returned from the statement variant of Describe,
-/// the format code is not yet known and will always be zero.
-public enum PostgresFormatCode: Int16, Codable, CustomStringConvertible {
+/// The format the postgres types are encoded in on the wire.
+///
+/// Currently there a two wire formats supported:
+///  - text
+///  - binary
+public enum PostgresFormat: Int16 {
```

Comment:
I wouldn't oppose adding it... However everything that is public API, will make binaries larger as it can't be stripped as long as we don't have lto. That's why I refrain from adding auto synthesized conformances normally. If I don't need it they are not allowed to take up bytes.